### ISSUE TITLE
Add Frames: support to Combined sub-sequences

### ISSUE THREAD

**Issue by reaperrr** [OpenRA/OpenRA#10469]
MechCommander sprites use one .shp file for each facing, so using `Combine` is a must.

However, probably to save memory or at least HDD space, they usually only cover half of the facing 'clock' and require considerable amounts of flipping and reverse-playing.
FlipX/Y are supported on `Combine`d sub-sequences, but `Frames` and `Reverse` are not. It would be extremely useful (and for some animations even necessary) to add at least `Frames` support to `Combine`d sub-sequences, to make move animations for turrets and barrels easier to implement.

Basically, I'd like our code to support something like this: 
https://gist.github.com/reaperrr/63900af4afe9a560e762

**Comment by reaperrr** [comment]
This would nicely complement the improvements of #17627, adding to Next+1 in hope someone gets around to implementing it soonish.

**Comment by pchote** [comment]
IMO we should implement a way to support this without having to define big `Combine` blocks, maybe something like:
```
myunit:
    idle:
        FilePattern: myunit-%04d.png
        FileStart: 0
        FileEnd: 15
        ...etc...
```

So I'd vote that we don't close this with #17723.

**Comment by reaperrr** [comment]
#17723 was good enough for immediate needs, so this isn't as urgent anymore. Moved to Future milestone (though I still hope we'll get this done sooner than some other stuff).

**Comment by pchote** [comment]
Expanding my thoughts above: we should move the custom `FrameSize` and `FrameAmount` embedded keywords from the `PngSheetLoader` into sequences too. There is no technical reason why we couldn't handle the sub-region slicing and SheetBuilder registration for this in the same place - although it will require a larger rework of the way the sequence loading works.
